src/main/drivers/pwm_output.h

```diff
@@ -29,6 +29,26 @@
 #endif
 
 typedef enum {
+    DSHOT_CMD_MOTOR_STOP = 0,
+    DSHOT_CMD_BEEP1,
+    DSHOT_CMD_BEEP2,
+    DSHOT_CMD_BEEP3,
+    DSHOT_CMD_BEEP4,
+    DSHOT_CMD_BEEP5,
+    DSHOT_CMD_ESC_INFO, //NOT USED
+    DSHOT_CMD_SPIN_ONE_WAY, //NOT USED
+    DSHOT_CMD_SPIN_OTHER_WAY, //NOT USED
+    DSHOT_CMD_3D_MODE_OFF, //NOT USED
+    DSHOT_CMD_3D_MODE_ON, //NOT USED
+    DSHOT_CMD_SETTINGS_REQUEST, //NOT USED
+    DSHOT_CMD_SAVE_SETTINGS, //NOT USED
```

Comment:
Just leave them in, without comments about implementation status in non-Betaflight. The definition for DShot commands includes all of the commands (except for 20 / 21, so maybe add a comment with respect to that). I think the Betaflight source is the wrong place to keep track of the implementation status of the commands in KISS / blheli_s / ..., the comments will just outdate and be confusing more than helpful.